example/helloWorld/src/helloWorld.cpp

```diff
@@ -133,12 +133,11 @@ auto example(TAccTag const&) -> int
     // Kernels can be everything that is trivially copyable, has a
     // callable operator() and takes the accelerator as first
     // argument. So a kernel can be a class or struct, a lambda, etc.
-    HelloWorldKernel helloWorldKernel;
+    alpaka::KernelBundle kernelBundle = HelloWorldKernel{};
```

Comment:
Shouldn't this be
```
alpaka::KernelBundle kernelBundle = alpaka::KernelBundle(HelloWorldKernel{});
```
?

Reply:
Ah, the explicit constructor call is not needed, because it is not marked as `explicit` ?

Reply:
`auto kernelBundle = alpaka::KernelBundle(HelloWorldKernel{});`

will be the long why to write the same line of code I did. The constructor is no marked `explicit` but all types will be deducted because the second parameter of the constructor is a variadic template.